### DISCUSSION TITLE
[Camera Support]: Does autotracking work with this camera: DH-IPC-HDW1230S?

### DISCUSSION THREAD

**Discussion by tamueller** [blakeblackshear/frigate#10965] [Camera Support]
### Describe the problem you are having

Hi,
I'd like to buy a camera <150€ that's capable of autotracking. Since only Dahua and Amcrest seem to support this correctly, but the 3 models listed in recommended hardware are too expensive or not available in europe I looked for a cheaper alternative from Dahua, and found this one: DH-IPC-HDW1230S

https://www.dahuasecurity.com/asset/upload/product/20180323/DH-IPC-HDW1230S_Datasheet_20180202.pdf

It says onvif and rtsp, and does have PoE which is nice for me. I could not find anything about this camera in frigate though, does anyone use it succesfully, especially with the autotracking feature?

### Version

n/a

### Frigate config file

```yaml
n/a
```

### Relevant log output

```shell
n/a
```

### FFprobe output from your camera

```shell
n/a
```

### Frigate stats

```json
n/a
```

### Operating system

HassOS

### Install method

HassOS Addon

### Coral version

USB

### Network connection

Wired

### Camera make and model

Dahua DH-IPC-HDW1230S

**Comment by NickM-27** [comment]
To me it seems unlikely that it would, nothing to indicate for sure though
